### ISSUE TITLE
'core' attribute is breaking the killboard

### ISSUE THREAD

**Issue by toconnell** [theLaborInVain/kdm-manager-api#74]
```
toconnell@mona:~/kdm-manager-api$ ./world.sh --debug killboard


Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/toconnell/kdm-manager-api/app/world/__main__.py", line 76, in <module>
    print(WORLD.debug_query(OPTIONS.debug_query))
  File "/home/toconnell/kdm-manager-api/app/world/__init__.py", line 394, in debug_query
    results = self.update_asset_dict(asset_dict)
  File "/home/toconnell/kdm-manager-api/app/world/__init__.py", line 271, in update_asset_dict
    fresh_results = update_method()
  File "/home/toconnell/kdm-manager-api/app/world/__init__.py", line 1032, in killboard
    killboard[m_asset["sub_type"]][m_asset['handle']] = {
KeyError: 'core'
```

**Comment by toconnell** [comment]
The logic problem here seems to be that we're getting our 'known' types from the killboard, rather than from looking at monster assets:

![image](https://github.com/theLaborInVain/kdm-manager-api/assets/48043/6965e74e-5537-43c2-95f6-4ab4eb81d426)

Later, when we start to format the output, we work from the monster assets:

![image](https://github.com/theLaborInVain/kdm-manager-api/assets/48043/35c428cc-70ef-4a7b-9d1a-3386ddcb835c)

**Comment by toconnell** [comment]
I think the fix here is to make the known types from the monster assets and then augment it with the ones in the database, correcting for ones with bad types.

**Comment by toconnell** [comment]
This is resolved in the 1.197.1226 hotfix that went into production last night.
